### DISCUSSION TITLE
I got unable to connect database error, how can i fix it ?

### DISCUSSION THREAD

**Discussion by Lambersonistaken** [brocoders/nestjs-boilerplate#1358] [Q&A] [answered]
![env - testtool - Visual Studio Code 1 02 2024 16_06_47](https://github.com/brocoders/nestjs-boilerplate/assets/73862428/b76eb48a-392a-40bd-9f97-018102ab48b5)

This is the error that i get. Any help appreciated, thank you.

**Comment by Shchepotin** [comment, marked as answer]
@Lambersonistaken did you run `docker compose -f docker-compose.document.yaml up -d mongo mongo-express maildev` ?
